### ISSUE TITLE
About initial learing rate

### ISSUE THREAD

**Issue by Marcusryz** [hyunwoongko/transformer#25]
I find that the initial learing rate in README is 0.1, but in conf.py is 1e-5. So which one is correct?

**Comment by Marcusryz** [comment]
Additionally, while I use the initial learning rate in conf.py(1e-5), the result of training is not the same with your image.

**Comment by Mithrosah** [comment]
> Additionally, while I use the initial learning rate in conf.py(1e-5), the result of training is not the same with your image.

Yeah. I use 1e-5 as initial learning rate and got a different result (all other params unchanged). After 1000 epochs, training loss is around 3.48, and test loss is around 3.42. And the final bleu score is around 23.4. Not pretty much the same as the author suggested.

Did u find out what's wrong, friend? I'm eager to know.
